src/components/addTask_form/addTask_form.html

```diff
@@ -0,0 +1,35 @@
+<div class="wrapper">
+  <form action="" class="form">
+    <section class="description">
+      <fieldset class="description">
+        <label for="description">Description </label>
+        <input class="enter" id="description" type="text" value="Enter Task" />
+      </fieldset>
+    </section>
+    <section>
+      <fieldset class="when">
+        <legend>when</legend>
+        <div class="date">
+          <label for="date_1">
+            <input id="today" type="radio" id="date_1" name="today" />
+            Today
+          </label>
+          <label for="date_2">
+            <input id="tomorrow" type="radio" id="date_2" name="tomorrow" />
+            Tomorrow
+          </label>
+          <label for="date_3">
+            <input
+              id="selectdate"
+              type="radio"
+              id="date_3"
+              name="select_date"
+            />
+            Select Date
+          </label>
+        </div>
+      </fieldset>
+    </section>
+    <button class="button" type="submit">Add Task</button>
```

Comment:
Every `button` element in a `form` is automatically `type="submit"`. I would also be more specific with the class name. If you follow BEM, it could be `form__button`.
```suggestion
    <button class="button">Add Task</button>
```